tests/conftest.py

```diff
@@ -12,6 +12,7 @@
 from typing import Any, Generator
 from unittest.mock import MagicMock, patch
 
+from _pytest.assertion.util import _compare_eq_iterable
```

Comment:
I think it's OK to call protected functions from tests, but we could copy the implementation from pytest if we don't want that